docs/contributing/contribution-checklists.md

```diff
@@ -42,7 +42,7 @@ In addition to the below checklist, please see the [Common Review
 Items](pullrequest-submission-and-lifecycle.md#common-review-items) sections for more specific coding and testing
 guidelines.
 
- - [ ] __Acceptance test coverage of new behavior__: Existing resources each
+- [ ] __Acceptance test coverage of new behavior__: Existing resources each
    have a set of [acceptance tests][acctests] covering their functionality.
```

Comment:
This may be a weird question, but does the overall formatting affect anything? Some sections appear to be wrapped, while others aren't. Asking for knowledge sake more than anything.

Reply:
Great question, @tracypholmes! Markdown tries to be nice and allow you to optionally wrap its source without actually inserting new lines into the formatted document. So really it is more of a Markdown style question for the folks reading and writing the Markdown source code, versus it affecting what the final document looks like.

We actually have some internal guidance on this subject added maybe in the last 6 months or so (let me know if you need a link) that says we should prefer to _not_ wrap text in Markdown source. I believe this preference is for a few reasons:

- Different maintainers may have different preferences on line length (instead they can setup their Markdown editor as they please)
- Changing the content of wrapped text is certainly more difficult/annoying as it can cause many line updates for simple changes
- Relating to the above, those changes make reviews much harder, since Git and GitHub cannot really make those differences more readable

At some point, we can go back through our existing Markdown source such as this documentation and fix it to be in line with those recommendations. 😄  Hopefully this helps answer your question.

Reply:
It did - thanks! I just wanted to make sure as I tend to wrap mine while working, but don't want it to affect anyone else or something like this pr. Appreciate you responding 😃